build.yaml

```diff
@@ -10,6 +10,9 @@ builders:
     builder_factories:
       - testHtmlBuilder
     build_extensions:
+      $test$:
+        - templates/default_template.html
+        - test_html_builder_config.json
       $package$:
         - test/templates/default_template.html
         - test/test_html_builder_config.json
```

Comment:
What is the effect of listing these files under both `$test$` and `$package$`? I would have expected we'd only need them listed under one or the other not both.

Reply:
Oh good catch, I think this was a bad merge conflict resolution!

Reply:
Should be fixed now